package.json

```diff
@@ -24,29 +24,33 @@
   "dependencies": {
     "@dooboo-ui/native": "^0.2.0",
     "@expo/vector-icons": "^10.0.6",
-    "@types/react-navigation": "^3.0.8",
     "expo": "^35.0.0",
     "expo-app-auth": "^7.0.0",
     "expo-constants": "^7.0.0",
     "expo-facebook": "^7.0.0",
     "expo-google-sign-in": "^7.0.0",
     "expo-linear-gradient": "~7.0.0",
     "expo-localization": "~7.0.0",
+    "formik": "^1.5.8",
     "i18n-js": "^3.3.0",
     "moment": "^2.24.0",
     "react": "16.8.6",
     "react-dom": "^16.8.6",
     "react-native": "https://github.com/expo/react-native/archive/sdk-35.0.0.tar.gz",
     "react-native-animatable": "^1.3.2",
-    "react-native-gesture-handler": "1.3.0",
+    "react-native-formik": "^1.7.7",
+    "react-native-gesture-handler": "~1.4.1",
```

Comment:
Please revert!

Reply:
refer to [here](https://github.com/dooboolab/hackatalk-mobile/pull/21/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R42). It's clean. You're indicating an outdated script's line. Please confirm and merge.